Pipfile

```diff
@@ -8,6 +8,8 @@ thoth-storages = ">=0.0.27"
 click = "*"
 thoth-common = ">=0.0.3"
 requests = "*"
+yarl = "<1.2"
```

Comment:
Please rebase and remove this as this is already handled by thoth-storages package that is the "user" of yarl - we should restrict dependencies in the package that uses it (sorry, my comment before was misleading here).